### PR TITLE
Introduce version 1.0.0

### PR DESCRIPTION
**Pull request by ondrej-hosak**
Add Any::Logger to module class
Add touch method for starting a teamcity job
Changed return interface of futures representing teamcity job

### CODE REVIEW

**Comment by ondrej-hosak** [comment]
please review @JaSei @tciml @luckamohelnikova

File: cpanfile [hunk 1]
```diff
@@ -8,6 +8,8 @@ requires 'Future';
 requires 'IO::Socket::SSL';
 requires 'IO::Async::Loop';
 requires 'IO::Async::Timer::Periodic';
+requires 'Log::Any::Adapter::Dispatch';
```
Comment (JaSei) [inline cpanfile]: I think this module isn't needed... Only in script which use this module...

**Comment by ondrej-hosak** [comment]
please review updated version

File: lib/Teamcity/Executor.pm [hunk 2]
```diff
@@ -48,49 +59,41 @@ has teamcity_auth_url => (
...
-
-        last if $response->{status} != 599;
-        print ' [TeamCity request retry: ' if !$retry;
-        print '.';
-        sleep 1;
-        $retry = 1;
+    # this code handles the teamcity authentification issues (sometimes authentification fails
+    # without a reason)
+    retry {
+      $response = $self->http->request(
```
[the remark this answers is not in the document]
Comment (ondrej-hosak) [reply lib/Teamcity/Executor.pm]: This is older version, see the newer commit. The code looks different.

**Pull request merged**